lib/libpmcstat/libpmcstat_image.c

```diff
@@ -120,14 +120,24 @@ pmcstat_image_add_symbols(struct pmcstat_image *image, Elf *e,
 		    == NULL)
 			continue;
 #ifdef __arm__
-		/* Remove spurious ARM function name. */
+		/* Ignore ARM mapping symbols. */
 		if (fnname[0] == '$' &&
 		    (fnname[1] == 'a' || fnname[1] == 't' ||
-		    fnname[1] == 'd') &&
-		    fnname[2] == '\0')
+		    fnname[1] == 'd' || ffname[1] == 'x' ||
+		    fnname[1] == 'c'))
 			continue;
 #endif
 
+#ifdef __aarch64__
```

Comment:
I guess technically we should upstream this for Thumb...

Reply:
Mmm, perhaps so.

Reply:
Have a version of this in my set of pending changes to upstream.